lib/mozart_fetcher/http_client.ex

```diff
@@ -1,39 +1,44 @@
 defmodule HTTPClient do
   require Logger
-  use ExMetrics
 
   alias MozartFetcher.TimeoutParser
 
   def get(endpoint, client \\ client()) do
     try do
-      ExMetrics.timeframe "function.timing.http_client.get" do
-        headers = set_request_headers(endpoint)
+      before_time = System.monotonic_time(:millisecond)
+      headers = set_request_headers(endpoint)
 
-        options = [
-          recv_timeout: TimeoutParser.parse(endpoint),
-          ssl: MozartFetcher.request_ssl(),
-          hackney: [pool: :origin_pool]
-        ]
+      options = [
+        recv_timeout: TimeoutParser.parse(endpoint),
+        ssl: MozartFetcher.request_ssl() ++ [verify: :verify_none],
```

Comment:
What about adding `[verify: :verify_none]` in `request_ssl` function instead of adding it here, what do you think?

Reply:
Yea I was thinking about that but then I decided not to, happy to move it in there. It has a test for this function so just a heads up with have a test change too.

Reply:
Done https://github.com/bbc/mozart-fetcher/pull/82/commits/7d13e1b1931622a223eafe5e8b4f910ef288d2d5